packages/block-library/src/post-comments-count/transforms.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+
+const transforms = {
+	to: [
+		{
+			type: 'block',
+			blocks: [ 'core/post-comments-link' ],
+			transform: ( attributes ) => {
+				return createBlock( 'core/post-comments-link', {
+					textAlign: attributes.textAlign,
+				} );
+			},
```

Comment:
```suggestion
			transform: ( { textAlign } ) => {
				return createBlock( 'core/post-comments-link', {
					textAlign,
				} );
			},
```

We can simplify the code by using two features of JavaScript:

- Object destructuring
- Object Property Shorthand